ob_intern/Controller/Edit Short Video Cover/EditShortVideoCoverViewController.swift

```diff
@@ -0,0 +1,114 @@
+//
+//  EditVideoTimelineViewController.swift
+//  ob_intern
+//
+//  Created by Soft Liampisan on 9/6/2566 BE.
+//  Copyright (c) 2566 BE ___ORGANIZATIONNAME___. All rights reserved.
+
+import UIKit
+import SDWebImage
+
+class EditShortVideoCoverViewController: UIViewController {
+
+    //MARK: - New Instance
+    class func newInstance() -> EditShortVideoCoverViewController {
+        let viewController = EditShortVideoCoverViewController(nibName: String(describing: EditShortVideoCoverViewController.self),
+                                                       bundle: nil)
+        
+        let viewModel = EditShortVideoCoverViewModel(delegate: viewController)
+        viewController.viewModel = viewModel
+        
+        return viewController
+    }
+    
+    //MARK: - IBOutlet
+    @IBOutlet weak var buttonCancel: UIButton!
+    @IBOutlet weak var buttonAgree: UIButton!
+    @IBOutlet weak var labelSelectCover: UILabel!
+    @IBOutlet weak var imageViewVideo: UIImageView!
+    @IBOutlet weak var collectionView: UICollectionView!
+    
+    //MARK: - Parameters
+    private var WIDTH: CGFloat = 0
+    private var HEIGHT: CGFloat = 0
+    private let NUM_ITEMS: CGFloat = 8
+    private var viewModel: EditShortVideoCoverViewModel?
+    
+    //MARK: - Lifecycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.setupView()
+        collectionView.reloadData()
+
+    }
+    
+    //MARK: - Functions
+    func setupView() {
+        imageViewVideo.layer.cornerRadius = 21
+        imageViewVideo.clipsToBounds = true
+        imageViewVideo.sd_setImage(with: URL(string: "https://i.pinimg.com/originals/31/9f/f9/319ff939cbca334407451fa12613783e.jpg"))
+        self.navigationController?.setNavigationBarHidden(true, animated: false)
+        collectionView.delegate = self
+        collectionView.dataSource = self
+        collectionView.register(UINib(nibName: String(describing: EditShortVideoCoverCollectionViewCell.self), bundle: nil), forCellWithReuseIdentifier: "EditShortVideoCoverCollectionViewCell")
```

Comment:
```suggestion
        let nibName = String(describing: EditShortVideoCoverCollectionViewCell.self)
        collectionView.register(UINib(nibName: nibName, bundle: nil), forCellWithReuseIdentifier: nibName)
```
สร้างเพื่อ initial string รอบเดียว แล้วก็ใช้ได้ทั้งหน้าและหลัง เพราะมันเหมือนกันครับ